README.md

```diff
@@ -99,13 +64,13 @@ After you're finished please remove all the comments and instructions!
 
 > Describe between 1-3 key features of the application.
 
```

Comment:
- [ ] I think it would better and more professional if you remove `Describe between 1-3 key features of the application.` as this is a guide for you and not for readers 👩‍⚖️ Templates, placeholders or boiler plates are meant to act as guide for our benefits, and important to customize and tailor these resources. I know you can do that 💪

**_Note: 👉 Please make sure that you apply this to entire README_**